lib/KIcon/iconDefinitions.js

```diff
@@ -69,6 +69,10 @@ const KolibriIcons = {
     defaultColor: themePalette().grey.v_400,
   },
   sort: { icon: require('./precompiled-icons/le/sort.vue').default },
+  sortColumn: {
+    icon: require('./precompiled-icons/le/sortColumn.vue').default,
+    defaultColor: themePalette().grey.v_400,
```

Comment:
@BabyElias Since this is quite a general icon, I'd recommend we don't override the default color here (dropdown up and down don't have it set either) and just keep it by default.

Instead, you can just set it on `KIcon`'s `color` prop in the table component.

Reply:
Done